designer/client/conditions/inline-conditions-definition.js

```diff
@@ -129,26 +132,28 @@ class InlineConditionsDefinition extends React.Component {
           </select>
 
           {fieldDef &&
-          <select className='govuk-select' id='cond-operator' name='cond-operator' value={condition.operator??''}
-            onChange={this.onChangeOperator}>
-            <option />
-            {
-              getOperatorNames(fieldDef.type).map(conditional => {
-                return <option key={`${condition.field}-${conditional}`}
-                  value={conditional}>{conditional}</option>
-              })
-            }
-          </select>
-          }
+            <select
+              className='govuk-select' id='cond-operator' name='cond-operator' value={condition.operator ?? ''}
+              onChange={this.onChangeOperator}
+            >
+              <option />
+              {
+                getOperatorNames(fieldDef.type).map(conditional => {
+                  return <option
+                    key={`${condition.field}-${conditional}`}
+                    value={conditional}
+                  >{conditional}
```

Comment:
Yikes! Is this what eslint demands? It comes out pretty hard to read!

Reply:
aye, we can always change the rules if it doesn't suit us. I'll have a look at what's causing it